lib/matplotlib/rcsetup.py

```diff
@@ -81,18 +81,19 @@ def __call__(self, s):
         raise ValueError(msg)
 
 
-def _listify_validator(scalar_validator, allow_stringlist=False, *, doc=None):
+@lru_cache()
```

Comment:
Does this get called so much that this cache has an effect?

Reply:
I doubt it really matters performance-wise, but e.g. validate_whiskers and validate_sketch create a new listified_validator every time they are called.